### PR TITLE
improve dx

### PR DESCRIPTION
**Pull request by cleemansen**

### CODE REVIEW

**Comment by cleemansen** [comment]
- [x] Docker just works

File: ShinyModule.R [hunk 1]
```diff
@@ -2,24 +2,39 @@ library("shiny")
...
+# one can use the function from the logger.R file:
+# logger.fatal(), logger.error(), logger.warn(), logger.info(), logger.debug(), logger.trace()
 
+shinyModuleUserInterface <- function(id, label, sdk, year) {
+  # all IDs of UI functions need to be wrapped in ns()
+  ns <- NS(id) 
+  # showcase to access a file ('auxiliary files') that is 
+  # a) provided by the app-developer and 
+  # b) can be overridden by the workflow user.
+  fileName <- paste0(getAppFilePath("yourLocalFileSettingId"), "sample.txt")
```
Comment (cleemansen) [inline ShinyModule.R:14]: I included a showcase how to use the "local file upload" feature.

image

Is this a good idea? If yes we should do the same for the _Shiny_ template, too.

Comment (cleemansen) [reply ShinyModule.R:14]: removed app-settings (was fully replaced by shiny bookmark!)

Comment (cleemansen) [reply ShinyModule.R:14]: Showcase "local file upload" is now:

image

**Review by annescharf — APPROVED** [review]

**Pull request merged**